otter/convergence/composition.py

```diff
@@ -86,3 +106,20 @@ def prepare_server_launch_config(group_id, server_config, lb_descriptions):
         generate_metadata(group_id, lb_descriptions))
 
     return set_in(server_config, ('server', 'metadata'), updated_metadata)
+
+
+def prepare_stack_launch_config(group_id, stack_config):
+    """
+    Prepare a stack config (the stack part of the Group's launch config)
+    with any necessary dynamic data.
+
+    :param str group_id: The group ID
+    :param PMap stack_config: The stack part of the Group's launch config,
+        as per :obj:`otter.json_schema.group_schemas.stack`.
+    """
+    # Set stack name and tag to the same thing
+    stack_config = set_in(stack_config, ('stack_name',),
+                          get_stack_tag_for_group(group_id))
+    stack_config = set_in(stack_config, ('tags',),
+                          get_stack_tag_for_group(group_id))
```

Comment:
Right now, there can't be because `stack_name` and `tags` are disallowed in the schema (see the JSON schema PR).